FreeSurfer/LongitudinalFreeSurferPipeline.sh

```diff
@@ -193,6 +141,99 @@ validate_freesurfer_version()
   fi
 }
 
+set -eu
```

Comment:
Why did you move this down?  I would prefer that the argument parsing be at the top of the script for easier reference.  If there is a line in the tool version code that is failing under `set -eu`, we should fix it (or if necessary, temporarily disable `set -eu`), not mess with the code order.

Reply:
I find it hard to parse code with functions intertwined with main code. But, no big deal, I moved it back.

Reply:
I find it harder to navigate if functions are not in the beginning of the file but rather intertwined with main code. But no big deal, I reversed it.